pkg/sidecar-controller/snapshot_controller.go

```diff
@@ -300,8 +300,8 @@ func (ctrl *csiSnapshotSideCarController) createSnapshotWrapper(content *crdv1.V
 		// storage system has responded with an error
 		klog.Infof("createSnapshotWrapper: CreateSnapshot for content %s returned error: %v", content.Name, err)
 		if isCSIFinalError(err) {
-			if err := ctrl.removeAnnVolumeSnapshotBeingCreated(content); err != nil {
-				return nil, fmt.Errorf("failed to remove VolumeSnapshotBeingCreated annotation from the content %s: %s", content.Name, err)
+			if remoteAnnotationErr := ctrl.removeAnnVolumeSnapshotBeingCreated(content); err != nil {
```

Comment:
remoteAnnotationErr -> removeAnnotationErr

err != nil -> remoteAnnotationErr != nil

Reply:
Apologies. This has been fixed.